File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/status_impressora.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class StatusImpressora extends TermicaCommand {
  final int _param;

  StatusImpressora(this._param) : super('StatusImpressora');

  @override
  functionParameters() => '"param"' ':' '$_param';
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/Commands/StatusImpressora.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class StatusImpressora(private val param: Int) : TermicaCommand("StatusImpressora") {
    override fun functionParameters(): String {
        return "\"param\"" + ":" + param
    }
}